Sources/TestsCommon/CommonTestUtils.swift

```diff
@@ -214,6 +303,25 @@ public func sortedEqual(_ expectedValue: BSONDocument?) -> Predicate<BSONDocumen
     }
 }
 
+/// Prints a message if a server version or topology requirement is not met and a test is skipped
+public func printRequirementNotMetMessage(
```

Comment:
I think `printSkipMessage` would be a bit briefer and more accurate name for this.

Reply:
ok done